lib/web_ui/lib/src/ui/painting.dart

```diff
@@ -1454,6 +1454,19 @@ class ImageFilter {
     //    if (matrix4.length != 16)
     //      throw ArgumentError('"matrix4" must have 16 entries.');
   }
+
+  ImageFilter.fromColorFilter(ColorFilter colorFilter) {
+     // TODO(flutter_web): add implementation.
+    throw UnimplementedError(
+        'ImageFilter.fromColorFilter not implemented for web platform.');
```

Comment:
I'll tag @yjbanov for a recommendation on whether this should be implemented here, or an Issue filed so it can be tracked and/or assigned...

(Same comment for the new `ImageFilter.compose()` method below...)